### PR TITLE
Tablet mode application

### PR DESCRIPTION
**Pull request by sgober**
_Pull requests into Flux require the following. Submitter should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:. Reviewers should complete every item on the bulleted list when reviewing._

## Submitter:
JIRA 1959, use endpoint `/poc` to test

**Running the Application**

- [x] Manually tested in Chrome (primary browser for testing)
- [x] Manually tested in IE/Safari (verify app loads and basic feature testing)
- [x] Manually tested in Chrome in iPad mode (at minimum, viewing patient record)

**Task**

- [x] Branch implements task as expected
- [ ] Key application features are updated as needed for task and tested (see: [Key Application Features](https://github.com/FluxNotes/flux/wiki/Key-Application-Features))
- [ ] Task specific definition of done (as described in the Jira task) is met, if applicable.

**Documentation**

- [x] This pull request describes why these changes were made
- [ ] Recognizes any potential shortcomings/bugs in the description 
- [x] Pre release script is updated on the [Pre Release Testing Script wiki page](https://github.com/FluxNotes/flux/wiki/Pre-Release-Testing-Script) (add/remove single features to test, do not run through full script)
- [ ] Update the list of key features with new features on the [Key Application Features wiki page](https://github.com/FluxNotes/flux/wiki/Key-Application-Features)
- [ ] Document any new APIs/extension points
- [ ] Additional technical components and libraries are documented and added to [wiki](https://github.com/FluxNotes/flux/wiki/About-Flux-Notes) as needed

**Code Quality**

- [ ] Code style guide is followed (see: [Code Style Guide](https://github.com/FluxNotes/flux/wiki/JavaScript-and-HTML-Code-Style-Guide))
- [ ] Code is commented

**Tests**

- [x] Existing tests passed
- [ ] Added backend tests for new functionality added

## Reviewers:

- Check that code is maintainable and reusable. Code reuses existing code and infrastructure where appropriate.
- Check the PR and code accomplishes the task's purpose.
- Check that new tests appropriately test the new code, including edge cases. Other existing tests pass.
- Verify Pull Request checklist is correctly completed by submitter.
- Try to break the code

### CODE REVIEW

File: src/containers/PointOfCareApp.jsx [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+import PointOfCareDashboard from '../dashboard/PointOfCareDashboard';
+import DataAccess from '../dataaccess/DataAccess';
+import SummaryMetadata from '../summary/SummaryMetadata';
+import PatientControlPanel from '../panels/PatientControlPanel';
+import PreferenceManager from '../preferences/PreferenceManager';
+import SearchIndex from '../patientControl/SearchIndex';
+import LoadingAnimation from '../loading/LoadingAnimation';
+import LoadingError from '../loading/LoadingError';
+
+import '../styles/PointOfCareApp.css'; // change me?
```
Comment (jafeltra) [inline src/containers/PointOfCareApp.jsx]: Do we need to still change this? Or can we get rid of the comment?

File: src/containers/PointOfCareApp.jsx [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+                            message={this.state.snackbarMessage}
+                        />
+                    </Grid>
+                </div>
+            </MuiThemeProvider>
+        );
+    }
+}
+
+// TO-DO: CHECK THESE THESE
```
Comment (jafeltra) [inline src/containers/PointOfCareApp.jsx]: Is this still outstanding?

File: src/dashboard/PointOfCareDashboard.jsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                        isAppBlurred={this.props.isAppBlurred}
+                        setAppBlur={this.props.setAppBlur}
+                    />
+                </div>
+            </div>
+        );
+    }
+}
+
+// TO-DO: CHECK ME
```
Comment (jafeltra) [inline src/dashboard/PointOfCareDashboard.jsx]: Is this still outstanding?

**Comment by sgober** [comment]
JA1, JA2 and JA4 are addressed.

For JA3, that makes sense to me, I definitely just copied and pasted the route.  @gregquinn2001 can you confirm this is what is wanted?

**Comment by gregquinn2001** [comment]
For JA3, I think Julia is correct, we should use the same clinician as pilot1 uses.

**Comment by sgober** [comment]
JA3 all set!

File: src/styles/PointOfCareApp.scss [hunk 4]
```diff
@@ -0,0 +1,71 @@
+@import './variables';
+
+.FullApp-content {
```
Comment (Dtphelan1) [inline src/styles/PointOfCareApp.scss]: We should really change this class and the associate className in the code to align with the POC naming convention.

**Comment by Dtphelan1** [comment]
Other than my minor comment above, I feel good about this. Great work Sgober! I'll approve after that's addressed.

**Comment by sgober** [comment]
DP1 all set!

**Pull request merged**